plugin/evm/atomic/state/atomic_trie_iterator_test.go

```diff
@@ -10,24 +10,19 @@ import (
 	"github.com/ava-labs/avalanchego/database/memdb"
 	"github.com/ava-labs/avalanchego/database/versiondb"
 	"github.com/ava-labs/avalanchego/ids"
-	"github.com/ava-labs/avalanchego/snow/snowtest"
-	"github.com/ava-labs/avalanchego/utils"
-	"github.com/ava-labs/coreth/plugin/evm/atomic"
+	avalancheutils "github.com/ava-labs/avalanchego/utils"
```

Comment:
why the rename? That causes seemingly unneeded diffs

Reply:
must be another `utils` during sometime